### PR TITLE
feat: Add panda-css integration for brisa

### PR DESCRIPTION
**Pull request by anusikh**
Related https://github.com/brisa-build/brisa/issues/516

Through this PR, i have added panda-css integration with brisa
- updated the docs
- added a package called brisa-pandacss containing the integration code
- added `with-pandacss` example to demonstrate usage of panda-css

[brisa-pandacss-ex.webm](https://github.com/user-attachments/assets/1ebf01d0-e40a-479f-9df2-f1d2ce23d596)

### CODE REVIEW

**Review by aralroca — COMMENTED** [review]
Wow! How could you do it at this speed? I freak...!!! thank you very much for your contribution! I'll check it calmly between today and tomorrow, here in Barcelona it's time to sleep. Welcome as a contributor! ❤️

**Comment by anusikh** [comment]
Well, i had actually started the work a couple of hours before i had requested your permission for contributing 😄 . Please take your time! Thanks!

File: examples/with-pandacss/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "module",
+  "scripts": {
+    "dev": "brisa dev",
+    "dev:debug": "brisa dev --debug",
+    "prepare": "panda codegen",
+    "build": "brisa build",
+    "start": "brisa start"
+  },
+  "dependencies": {
+    "brisa": "0.1.0"
```
Comment (aralroca) [inline examples/with-pandacss/package.json:13]: I think you miss the dependency

```suggestion
    "brisa": "0.1.0",
    "@pandacss/dev": "latest"
```

File: examples/with-pandacss/brisa.config.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import type { Configuration } from 'brisa';
+import PandaCSS from 'brisa-PandaCSS';
```
Comment (aralroca) [inline examples/with-pandacss/brisa.config.ts]: just a nit

```suggestion
import PandaCSS from 'brisa-pandacss';
```

**Review by aralroca — APPROVED** [review]
looks very good, thank you very much for your contribution @anusikh ! If you write to contact@brisa.build with your address and t-shirt size, we will send you the Brisa t-shirt as a thank you!

**Comment by aralroca** [comment]
I see that by changing the name of the branch from “main” to “canary” I had failed to change the github action and that is why the tests in the PRs do not pass. But I have tried the tests in your fork and everything is great. Very good work.

**Comment by anusikh** [comment]
thanks for the info! I was a bit worried about the tests but that clears it!

**Pull request merged**